sunyar-fabric.md

```diff
@@ -26,4 +24,5 @@ _Provide the name of your sponsor. A sponsor is optional, but the sponsor must b
 
```

Comment:
We need names of the people who will be contributing to this lab. Without that, no one will have write access to commit to the GitHub repo.